### PR TITLE
[forge] Automatically generate a useful namespace

### PR DESCRIPTION
**Pull request by perryjrandall**
Make --namespace optional and give the default a somewhat easy to remember name

Test Plan: unittests

### CODE REVIEW

File: testsuite/forge-cli/src/main.rs [hunk 1]
```diff
@@ -259,11 +274,24 @@ fn main() -> Result<()> {
                     if let Some(move_modules_dir) = &k8s.move_modules_dir {
                         test_suite = test_suite.with_genesis_modules_path(move_modules_dir.clone());
                     }
+                    let namespace = if k8s.namespace.is_none() {
+                        let mut rng : ThreadRng= rand::thread_rng();
+                        let words = std::fs::File::open("/usr/share/dict/words")
```
Comment (sionescu) [inline testsuite/forge-cli/src/main.rs:279]: It probably makes sense to hard-code a sanitized list of words, and append a short (6-8 chars) random string. The `words` dict has plenty of NSFW words.

Comment (perryjrandall) [reply testsuite/forge-cli/src/main.rs:279]: I personally dont see a huge problem with that, however I can pick words of a certain length

File: testsuite/forge-cli/src/main.rs [hunk 2]
```diff
@@ -866,7 +895,7 @@ fn three_region_sim_graceful_overload(config: ForgeConfig) -> ForgeConfig {
                 // Additionally - we are not really gracefully handling overlaods,
                 // setting limits based on current reality, to make sure they
                 // don't regress, but something to investigate
-                avg_tps: 1200,
+                avg_tps: 1400,
```
Comment (rustielin) [inline testsuite/forge-cli/src/main.rs:898]: accident?

**Review by rustielin — APPROVED** [review]
LGTM! Thanks for making forge better tested and documented :) 

One small comment on (accidental?) bumping of success criteria

File: testsuite/forge-cli/src/main.rs (hunk 2 above)
Comment (perryjrandall) [reply testsuite/forge-cli/src/main.rs:898]: How the fuck...? Yes accident

File: testsuite/forge-cli/src/main.rs (hunk 1 above)
Comment (geekflyer) [reply testsuite/forge-cli/src/main.rs:280]: does this work across OS?

**Review by geekflyer — APPROVED** [review]
approving for now, but I must wonder - maybe it's better to use a library for the random name generation (to handle OS differences and whatnot)?

**Comment by perryjrandall** [comment]
> approving for now, but I must wonder - maybe it's better to use a library for the random name generation (to handle OS differences and whatnot)?

So I checked out random_words and actually that has a built in set of words and some nice convenience functions, thanks for the suggestion!

**Pull request merged**